geode-core/src/main/java/org/apache/geode/management/internal/configuration/mutators/RegionConfigManager.java

```diff
@@ -78,7 +78,7 @@ public void delete(RegionConfig config, CacheConfig existing) {
     for (RegionConfig config : staticRegionConfigs) {
       DistributedRegionMXBean distributedRegionMXBean =
           getManagementService().getDistributedRegionMXBean("/" + config.getName());
-      long entryCount = -1;
+      long entryCount = 0;
```

Comment:
I guess it all depends on if we want the user to know the difference between the situation when entrycount is actually 0 and the situation when the region is not fully initialized. I guess the interval is so trivial that we can erase the difference.